server/publish.js

```diff
@@ -1,3 +1,9 @@
-Meteor.publish('invoices', function(date, days) {
-  return InvoiceTicketsCollection.find(getFilter(date, days));
+Meteor.publish('invoices', function(date, days, limit) {
+  limit = limit || 10;
+  if (limit > 0) {
+    return InvoiceTicketsCollection.find(getFilter(date, days), {limit: limit});
+  }
+  else {
+  	return InvoiceTicketsCollection.find(getFilter(date, days));
```

Comment:
Not sure if we ever want to publish `all` invoices, in reality we probably would rather need an upper limit – again imagine you have millions of invoices, one subscription loading all invoices would probably be enough to crash your server & client.